pyscf/pbc/df/rsdf_jk.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# Copyright 2014-2020 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Hong-Zhou Ye <hzyechem@gmail.com>
+#
+
+import numpy
+import copy
+
+def density_fit(mf, auxbasis=None, mesh=None, with_df=None):
+    '''Generte density-fitting SCF object
```

Comment:
Generate

Reply:
Copied from `df_jk.py` so I will leave this feature typo for now.

Reply:
fixed